tools/local-presubmit

```diff
@@ -19,12 +19,6 @@ set -x
 
 ./tools/sigh webpack
 ./tools/bazelisk ${BAZELRC_OPTS} test //java/... //javatests/... //src/... //particles/...
-
-# Build apks used by Firebase Test Labs and copy them to /workspace which
```

Comment:
Ideally we would still run the Android e2e tests as part of `local-presubmit`, are they run by some other part of the script still?

Reply:
Yeah, they are moved into a different step so they can run parallel to local-presubmit.

Reply:
I'm missing something. Where are they moved to? If I am on my workstation and I want to run all of the tests on my workstation that run on GCB, the intent is that I can do that by runnign `local-presubmit`.

I think we are mixing things up -- @bgogul had a TODO to rename or rework GCB so that it doesn't use `local-presubmit` on the cloud. It's intended for local use (it's in the name).

So, splitting local-presubmit so that it does the right thing we want in cloud, but no longer does its original goal locally (replicate what the cloud run does), achieves an improvement to cloud but removes/breaks the goal of the script for local use.

Reply:
Perhaps a solution is to fork the script to say a `cloud-presubmit` script, and leave this `local-presubmit` as is. Or, we could add flags to it and on cloud we run it one way, and locally, another. Or, break out what's done here into cloudbuild.yaml commands more granularly.

I still feel like if we're using this on GCB we need to rename it or otherwise rework things, `local-presubmit` is intended for use locally, taking shortcuts if needed since it's just aiming to get as close to what happens on CI as possible.

Reply:
This is a naming issue then: I thought local-presubmit is the presubmit with a weird name. The solution is easy: I'll create a cloud-presubmit script so it can be used by cloud build.

Reply:
Oh we proposed the same name, nice.